src/server/server.ts

```diff
@@ -47,6 +47,15 @@ function isOfficeHost(host: string) {
     return officeHosts.indexOf(host) >= 0;
 }
 
+let assetPaths = null;
+function getAssetPaths() {
+    if (assetPaths === null) {
+        let data = fs.readFileSync(path.resolve(__dirname, 'assets.json'));
+        assetPaths = JSON.parse(data.toString());
+    }
+
+    return assetPaths;
+}
 
```

Comment:
Yes if the file read intermittently fails when the server starts up the static could be stuck in a bad state until iis reset.